arm-graphrbac/1.6/swagger/graphrbac.json

```diff
@@ -1068,15 +1360,15 @@
         },
         "usage": {
           "type": "string",
-          "description": "Gets or sets usage"
+          "description": "Gets or sets usage. Acceptable values are 'Verify' and 'Sign'."
```

Comment:
Removed "Gets or sets".
I didnt keep an enum for "Verify" and "AsymmetricX509Cert" because existing application can have a credential that is not set to these values and then the patch operation of those credentials will fail.